### PR TITLE
Remove period punctuation marks from copy in test app

### PR DESCRIPTION
**Pull request by dawiddylowicz**
# Problem
In test app, we had inconsistency in some copies ending will full stop.
This was reported by Applause here: https://platform.applause.com/products/19993/testcycles/183654/issues/3255767

# Solution
Remove the period punctuation marks from test app

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://335-pr-onfido-sdk-ui-onfido.surge.sh

File: src/locales/en.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 export const en = {
   welcome: {
     title: 'Open your new bank account',
-    description_p_1: 'To open a bank account, we will need to verify your identity.',
-    description_p_2: 'It will only take a couple of minutes.',
+    description_p_1: 'To open a bank account, we will need to verify your identity',
```
Comment (stefaniacardenas) [inline src/locales/en.js]: Does the Spanish copy have the right punctuation?

Comment (dawiddylowicz) [reply src/locales/en.js]: Yes you were right, pushed changes also for Spanish

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://335-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-335-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request closed without merging**